.gitignore

```diff
@@ -13,6 +13,9 @@ dist
 coverage
 .env
 
+# README is copied from root on build
+packages/openapi-ts/README.md
```

Comment:
@jordanshatford huh, this doesn't actually work for me. I am unable to gitignore README in package, are you able to resolve this?